### PR TITLE
[bn] Localization for DevSecOps

### PR DESCRIPTION
**Pull request by abhijeet-26**
Signed-off-by: Abhijeet Basfore abhibas14@gmail.com

### Describe your changes
link - > https://glossary.cncf.io/devsecops/

### Related issue number or link (ex: `resolves #issue-number`)
#410

### Checklist before opening this PR (put `x` in the checkboxes)
- [ x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by mitul3737 — CHANGES_REQUESTED** [review]
You have not added the details:
---
title: DevSecOps 
status: Completed
category: concept
tags: ["methodology", "security", ""]
---

in localized form

File: content/bn/devsecops.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## এটা কি
+
+দেভসেকঅপ্স (DevSecOps) শব্দটি উন্নয়ন, কর্মক্ষম, এবং নিরাপত্তা দায়িত্বের সাংস্কৃতিক একীকরণকে বোঝায়।
```
Comment (mitul3737) [inline content/bn/devsecops.md:3]:
```suggestion
ডেভসেকঅপ্স (DevSecOps) শব্দটি উন্নয়ন, কর্মক্ষম, এবং নিরাপত্তা দায়িত্বের সাংস্কৃতিক একীকরণকে বোঝায়।
```

File: content/bn/devsecops.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+এটি ডেভেলপার এবং অপারেশনাল ওয়ার্কফ্লোতে ন্যূনতম থেকে কোনও ব্যাঘাত না করে সুরক্ষা অগ্রাধিকারগুলি অন্তর্ভুক্ত করতে দেভঅপ্স [(DevOps)](/bn/devops/) পদ্ধতির প্রসারিত করে৷
```
Comment (mitul3737) [inline content/bn/devsecops.md:4]:
```suggestion
এটি ডেভেলপার এবং অপারেশনাল ওয়ার্কফ্লোতে ন্যূনতম  কোনও ব্যাঘাত না করে সুরক্ষা অগ্রাধিকারগুলি অন্তর্ভুক্ত করতে ডেভঅপ্স [(DevOps)](/bn/devops/) পদ্ধতির প্রসারিত করে৷ 
```

File: content/bn/devsecops.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## এটা কি
+
+দেভসেকঅপ্স (DevSecOps) শব্দটি উন্নয়ন, কর্মক্ষম, এবং নিরাপত্তা দায়িত্বের সাংস্কৃতিক একীকরণকে বোঝায়।
+এটি ডেভেলপার এবং অপারেশনাল ওয়ার্কফ্লোতে ন্যূনতম থেকে কোনও ব্যাঘাত না করে সুরক্ষা অগ্রাধিকারগুলি অন্তর্ভুক্ত করতে দেভঅপ্স [(DevOps)](/bn/devops/) পদ্ধতির প্রসারিত করে৷ 
+দেভঅপ্স-এর মতো, দেভসেকঅপ্স হল একটি সাংস্কৃতিক পরিবর্তন, যা গৃহীত প্রযুক্তির দ্বারা ঠেলে দেওয়া হয়, অনন্য গ্রহণের পদ্ধতি সহ।
```
Comment (mitul3737) [inline content/bn/devsecops.md:5]:
```suggestion
ডেভঅপ্স-এর মতো, ডেভসেকঅপ্স হল একটি সাংস্কৃতিক পরিবর্তন, যা গৃহীত প্রযুক্তির দ্বারা ঠেলে দেওয়া হয়, অনন্য গ্রহণের পদ্ধতি সহ।
```

File: content/bn/devsecops.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+দেভঅপ্স অনুশীলনের মধ্যে রয়েছে অবিচ্ছিন্ন একীকরণ [(continuous integration)](/bn/continuous-integration/) এবং
```
Comment (mitul3737) [inline content/bn/devsecops.md:9]:
```suggestion
ডেভঅপ্স অনুশীলনের মধ্যে রয়েছে অবিচ্ছিন্ন একীকরণ [(continuous integration)](/bn/continuous-integration/) এবং 
```

File: content/bn/devsecops.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+দেভঅপ্স অনুশীলনের মধ্যে রয়েছে অবিচ্ছিন্ন একীকরণ [(continuous integration)](/bn/continuous-integration/) এবং 
+অবিচ্ছিন্ন স্থাপনা [(continuous deployment)](/continuous-delivery/)  এবং অ্যাপ্লিকেশন বিকাশ এবং প্রকাশের চক্রকে ত্বরান্বিত করা। 
+দুর্ভাগ্যবশত, স্বয়ংক্রিয় রিলিজ প্রক্রিয়া যা সমস্ত সাংগঠনিক স্টেকহোল্ডারদের যথাযথভাবে প্রতিনিধিত্ব করতে ব্যর্থ হয় তা বিদ্যমান সমস্যাগুলিকে আরও বাড়িয়ে তুলতে পারে। 
+একটি প্রক্রিয়া যা নিরাপত্তার প্রয়োজন বিবেচনা না করে দ্রুত নতুন সফ্টওয়্যার প্রকাশ করে তা একটি প্রতিষ্ঠানের নিরাপত্তা ভঙ্গিকে অবনমিত করতে পারে। 
+
+## এটা কিভাবে সাহায্য করে
+
+দেভসেকঅপ্স টিম সাইলো ভাঙ্গার উপর ফোকাস করে এবং সুরক্ষিত, স্বয়ংক্রিয় ওয়ার্কফ্লো তৈরির প্রচার করে।
```
Comment (mitul3737) [inline content/bn/devsecops.md:16]:
```suggestion
ডেভসেকঅপ্স টিম সাইলো (silos) ভাঙ্গার উপর ফোকাস করে এবং সুরক্ষিত, স্বয়ংক্রিয় ওয়ার্কফ্লো তৈরির প্রচার করে। 
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by Imtiaz1234 — APPROVED** [review]

**Pull request merged**